static/css/style.css

```diff
@@ -5,12 +5,18 @@
   box-sizing: border-box;
   font-family: "quicksand", sans-serif;
 }
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```

Comment:
unsure about these for the whole of the body? But might work

Reply:
Thats to get the footer to stick to the bottom on all pages